lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2202,6 +2202,12 @@ namespace {
                                    MoveOnlyAttr(/*Implicit=*/true));
       }
 
+      if (Impl.SwiftContext.LangOpts.hasFeature(Feature::NonescapableTypes) &&
```

Comment:
I did not want to change `CxxRecordSemantics::evaluate`, because I think `escapability` is orthogonal to many of the things there, like if a type is move only or not. (E.g., we could have move-only non-escapable types.) I think `CxxRecordSemantics` might need some rework in the future to account for orthogonality.

Reply:
Yeah, `CxxRecordSemantics` currently doesn't actually represent the semantics of the type. I agree it would need rework in the future.